test/deploy/validate-configs.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+# Script to validate the packaged module and configs
+
+# Ensure our workdir is that of the project root
+cd "${0%/*}/../.." || exit 
+
+TEST_NAME="Deployment testing"
+declare -a CONFIG_ARRAY
+if [[ $# -gt 0 ]]; then
+    for CONFIG in "$@"; do
+      CONFIG=$(ls {"$CONFIG",config/"$CONFIG",config/"$CONFIG".json,"$CONFIG".json} 2> /dev/null)
```

Comment:
If I understand correctly this means I can call this script with `config/file.json`, `file.json`, `file` and `config/file` as valid input parameters all corresponding to `config/file.json`? Would only pick one of those as a valid option. Personally I would require the "full" path so `config/file.json` for least confusion.

Reply:
Your assumption is correct. I'll change it back to only full path, might get confusing indeed.